app/models/likes.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Basically this method is supposed to increment the counter for the likes on a post for a specific user. This can easily be achieved by making use of the ActiveRecord increment!. Kindly consider using this. Have a look at [this](https://www.rubydoc.info/docs/rails/ActiveRecord%2FPersistence:increment!) or [this](https://stackoverflow.com/questions/11499110/increment-vs-1) for inspiration. 

**You will see how shorter this line will become** :smile:

Reply:
Ok, @Nemwel-Boniface Done ✔️ , thank you so much for taking the time to review my project.

Reply:
Eres muy bienvenido @angeluray